plugins/multinet.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: multinet
+spec:
+  version: v0.2.1
+  homepage: https://github.com/k8snetworkplumbingwg/kubectl-multinet
+  shortDescription: "Shows pods' network-status of multi-net-spec"
+  description: |
+    Shows pods' network-status annotation, defined in NPWG multi-net-spec, https://github.com/k8snetworkplumbingwg/multi-net-spec.
```

Comment:
Please wrap around at ~80 chars, so that it prints nicely on wide and narrow terminals.

Reply:
fixed. thanks.